dhis-2/dhis-services/dhis-service-tracker/pom.xml

```diff
@@ -141,6 +141,10 @@
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Is this because of `Enrollments` class?
If it is and if we agree that `Enrollments` class will disappear at some point, I will probably remove XML annotations from there and keep the pom clean

Reply:
yes it is and yes we will remove `Enrollments` at the end